quadriga/src/main/java/edu/asu/diging/quadriga/core/service/impl/CollectionManagerImpl.java

```diff
@@ -68,5 +68,21 @@ public Collection editCollection(String id, String name, String description) thr
             throw new CollectionNotFoundException();
         }
     }
+    
+    /**
+     * Deletes a collection from mongodb collection table by _id
+     * 
+     * @param id used to look up the collection in mongodb
+     * 
+     */
+    public void deleteCollection(String id) throws CollectionNotFoundException {
+        Collection collection = findCollection(id);
+        
+        if(Objects.nonNull(collection)) {
+            collectionRepo.delete(collection);
```

Comment:
let's make a comment here that.  this will have to be adjusted once there are networks in the collection.